### ISSUE TITLE
exfat-dkms dkms build fails for kernel version 5.2 and upwards

### ISSUE THREAD

**Issue by alpha-tre** [void-linux/void-packages#16203]
### System
* xuname:  
*Void 5.3.9_1 x86_64 AuthenticAMD uptodate rrrmFFFF*

* package:  
  *exfat-dkms version 1.2.8*

### Expected behavior
The dkms build process finishes succesfully
### Actual behavior
`exfat-dkms-1.2.8_5: configuring ...
DKMS module 'exfat-1.2.8' already added, skipping.
Building DKMS module 'exfat-1.2.8' for kernel-5.2.21_1... FAILED!
DKMS module 'exfat-1.2.8' failed to build, please check /var/lib/dkms
for errors in the log file.
Building DKMS module 'exfat-1.2.8' for kernel-5.3.6_1... FAILED!
DKMS module 'exfat-1.2.8' failed to build, please check /var/lib/dkms
for errors in the log file.
Building DKMS module 'exfat-1.2.8' for kernel-5.3.7_1... FAILED!
DKMS module 'exfat-1.2.8' failed to build, please check /var/lib/dkms
for errors in the log file.
Building DKMS module 'exfat-1.2.8' for kernel-5.3.8_1... FAILED!
DKMS module 'exfat-1.2.8' failed to build, please check /var/lib/dkms
for errors in the log file.
Building DKMS module 'exfat-1.2.8' for kernel-5.3.8_2... FAILED!
DKMS module 'exfat-1.2.8' failed to build, please check /var/lib/dkms
for errors in the log file.
Building DKMS module 'exfat-1.2.8' for kernel-5.3.9_1... FAILED!
DKMS module 'exfat-1.2.8' failed to build, please check /var/lib/dkms
for errors in the log file.
exfat-dkms-1.2.8_5: installed successfully.`

`DKMS make.log for exfat-1.2.8 for kernel 5.3.9_1 (x86_64)
Do 7. Nov 13:44:09 CET 2019
make -C /lib/modules/5.3.9_1/build M=/var/lib/dkms/exfat/1.2.8/build modules
make[1]: Entering directory '/usr/src/kernel-headers-5.3.9_1'
  CC [M]  /var/lib/dkms/exfat/1.2.8/build/exfat_core.o
/var/lib/dkms/exfat/1.2.8/build/exfat_core.c: In function 'fs_error':
/var/lib/dkms/exfat/1.2.8/build/exfat_core.c:1760:64: error: 'MS_RDONLY' undeclared (first use in this function); did you mean 'IS_RDONLY'?
 1760 |  else if ((opts->errors == EXFAT_ERRORS_RO) && !(sb->s_flags & MS_RDONLY)) {
      |                                                                ^~~~~~~~~
      |                                                                IS_RDONLY
/var/lib/dkms/exfat/1.2.8/build/exfat_core.c:1760:64: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [scripts/Makefile.build:281: /var/lib/dkms/exfat/1.2.8/build/exfat_core.o] Error 1
make[1]: *** [Makefile:1626: _module_/var/lib/dkms/exfat/1.2.8/build] Error 2
make[1]: Leaving directory '/usr/src/kernel-headers-5.3.9_1'
make: *** [Makefile:35: all] Error 2`
### Steps to reproduce the behavior
Try to install exfat-dkms on a system with the kernel version 5.2 and upwards

**Comment by Hoshpak** [comment]
duplicate of #14813

**Comment by alpha-tre** [comment]
@Hoshpak sorry i forgot to search for a preexisting issue.
